<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationTabWidgetBase</class>
 <widget class="QWidget" name="StationTabWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="openStnPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Open observation sites...</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Img/icons.qrc">
         <normaloff>:/Images/open-icon.png</normaloff>:/Images/open-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveStnPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Save observation sites...</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Img/icons.qrc">
         <normaloff>:/Images/save-icon.png</normaloff>:/Images/save-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeStnPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Remove observation sites</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Img/icons.qrc">
         <normaloff>:/Images/delete-icon.png</normaloff>:/Images/delete-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="StationTreeView" name="treeView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationTreeView</class>
   <extends>QTreeView</extends>
   <header>StationTreeView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>openStnPushButton</tabstop>
  <tabstop>saveStnPushButton</tabstop>
  <tabstop>removeStnPushButton</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources>
  <include location="../Img/icons.qrc"/>
 </resources>
 <connections/>
</ui>
